R/database.R

```diff
@@ -522,10 +522,10 @@ get_merged_result = function(tool_name,
                              projection = "grch37",
                              seq_type = "genome"){
 
```

Comment:
Is this function actually used by any other functions or should it be deprecated entirely?

Reply:
Indeed, this function was only called by `get_manta_sv()`. I have now removed it from GAMBLR.